azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/calling/participantlist/ParticipantListView.kt

```diff
@@ -161,16 +161,27 @@ internal class ParticipantListView(
         isMuted: Boolean,
         personaData: CommunicationUIPersonaData?,
     ): BottomCellItem {
+        val micIcon = if (isMuted) ContextCompat.getDrawable(
+            context,
+            R.drawable.azure_communication_ui_ic_fluent_mic_off_24_filled_composite_button_filled_grey
+        )
+        else ContextCompat.getDrawable(
+            context,
+            R.drawable.azure_communication_ui_ic_fluent_mic_on_24_filled_composite_button_filled_grey
+        )
+
+        val micAccessibilityAnnouncement = if (isMuted) context.getString(
+            R.string.azure_communication_ui_calling_view_participant_list_muted_accessibility_label
+        )
+        else "Unmuted"
```

Comment:
please use referenced string from string.xml

Reply:
Thanks. I was waiting for the pipeline translations to be made.